Makefile

```diff
@@ -1,37 +1,38 @@
+# use this to start docker + debugger 
```

Comment:
because mysql takes around 10 secs to run in docker, I had to split make debug to only restart the app without restarting everything, to keep development loop fast.
so make debug no longer runs everything, but if you type `make` alone it will but it takes longer time now